### PR TITLE
eBPF use of VXLAN MTU settings.

### PR DESCRIPTION
**Pull request by tomastigera**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 860e393b69b375db39ea2be4232ad4c2957d144e

https://deploy-preview-3459--calico.netlify.app

**Comment by tmjd** [comment]
Please change the `docs-pr-required` on https://github.com/projectcalico/calico/pull/3459 to `docs-completed` when this PR is merged.

File: reference/felix/configuration.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ The full list of parameters which can be set is as follows.
 | `UsageReportingInitialDelaySecs`  | `FELIX_USAGEREPORTINGINITIALDELAYSECS`  | Minimum delay before first usage report, in seconds. [Default: `300`] | int |
 | `UsageReportingIntervalSecs`      | `FELIX_USAGEREPORTINGINTERVALSECS`      | Interval at which to make usage reports, in seconds. [Default: `86400`] | int |
 | `VXLANEnabled`                    | `FELIX_VXLANENABLED`                    | Automatically set when needed, you shouldn't need to change this setting: whether Felix should create the VXLAN tunnel device for VXLAN networking. [Default: `false`] | boolean |
-| `VXLANMTU`                        | `FELIX_VXLANMTU`                        | The MTU to set on the VXLAN tunnel device. See [Configuring MTU]({{ site.baseurl }}/networking/mtu) [Default: `1410`] | int |
+| `VXLANMTU`                        | `FELIX_VXLANMTU`                        | The MTU to set on the VXLAN tunnel device. Also controls NodePort MTU when eBPF enabled. See [Configuring MTU]({{ site.baseurl }}/networking/mtu) [Default: `1410`] | int |
```
Comment (fasaxc) [inline reference/felix/configuration.md:68]: You should also update the felix configuration resource page (it's basically the same page but it describes the YAML format of the config).

**Comment by fasaxc** [comment]
Also, this PR adds MTU to the howto guide: https://github.com/projectcalico/calico/pull/3462

(Was adding something else in there and spotted MTU was still documented as a limitation.)

File: reference/felix/configuration.md (hunk 1 above)
Comment (tomastigera) [reply reference/felix/configuration.md:68]: done

**Pull request merged**